src/styles/createGenerateClassName.js

```diff
@@ -10,7 +10,7 @@ import type { Rule, generateClassName } from 'jss/lib/types';
 //
 // It's an improved version of
 // https://github.com/cssinjs/jss/blob/4e6a05dd3f7b6572fdd3ab216861d9e446c20331/src/utils/createGenerateClassName.js
-export default function createGenerateClassName(): generateClassName {
+export default function createGenerateClassName(prefix: string = "c"): generateClassName {
```

Comment:
@kgregory thank you. I'm planing to clone this to local when I have time and I test. By the way, when I see this project is using prettier, I assume that it uses the default config !